app/views/hiring_staff/vacancies/edit.html.haml

```diff
@@ -0,0 +1,92 @@
+%h1.heading-large
+  = t('vacancies.edit_heading', school: @vacancy.school.name)
+
```

Comment:
Do we need some way for the user to cancel and return to the `schools#show` page?

Reply:
@gazaston yes. Do you have any suggestions?

Reply:
Probably just a simple link above the `h1`?

Reply:
@gazaston done! After this PR is in we should maybe address navigation more generally. E.g. review page -> school page, vacancy#show -> school page etc.

Reply:
I'll merge this in as-is, but agree that we need to tackle this nav pattern. I'll address that immediately.